### PR TITLE
Move ToggleButton and ToggleButtonGroup to core

### PR DESCRIPTION
**Pull request by joshwooding**
The Toggle Button Group allows users to make a mutually exclusive selection from a set of related commands—with only one option selected at a time.
This Toggle Button allows users to enable or disable a single command.

```tsx
<ToggleButtonGroup>
  <ToggleButton value="all">
    <AppSwitcherIcon aria-hidden />
    All
  </ToggleButton>
  <ToggleButton value="active">
    <VisibleIcon aria-hidden />
    Active
  </ToggleButton>
  <ToggleButton disabled value="search">
    <FolderClosedIcon aria-hidden />
    Archived
  </ToggleButton>
</ToggleButtonGroup>
```

### CODE REVIEW

File: packages/core/stories/toggle-button-group/toggle-button-group.stories.tsx [hunk 1]
```diff
@@ -7,7 +7,7 @@ import {
   VisibleIcon,
 } from "@salt-ds/icons";
 import { ComponentMeta, ComponentStory } from "@storybook/react";
-import { Tooltip } from "@salt-ds/core";
+import { Tooltip } from "../../src";
```
Comment (origami-z) [inline packages/core/stories/toggle-button-group/toggle-button-group.stories.tsx:10]: Should this in the same `core` import group?

File: packages/core/stories/toggle-button/toggle-button.doc.stories.mdx [hunk 2]
```diff
@@ -23,37 +23,37 @@ This basic component allows users to enable or disable a single command.
 
 Use icons and text in Toggle Buttons if it's important to visually reinforce the theme of the options or if providing additional context may help decision-making (for examples, selecting a document type or a text style).
 
-- Pass `aria-hidden=”true”` to the icon so its label is not announced by the screen reader.
+- Pass `aria-hidden=”true”` to the icon so its coreel is not announced by the screen reader.
```
Comment (origami-z) [inline packages/core/stories/toggle-button/toggle-button.doc.stories.mdx:26]: need undo

File: packages/core/stories/toggle-button/toggle-button.doc.stories.mdx [hunk 3]
```diff
@@ -23,37 +23,37 @@ This basic component allows users to enable or disable a single command.
...
-  <Story id="lab-toggle-button--text-and-icon" />
+  <Story id="core-toggle-button--text-and-icon" />
 </Canvas>
 
 ### Icon only
 
 The Toggle Button Group can be used with icons only.
 
-- An accessible label should be provided for each button.
+- An accessible coreel should be provided for each button.
```
Comment (origami-z) [inline packages/core/stories/toggle-button/toggle-button.doc.stories.mdx:36]: undo

File: packages/lab/src/query-input/internal/QueryInputBody.tsx [hunk 4]
```diff
@@ -10,8 +10,8 @@ import {
   useMemo,
 } from "react";
 
-import { ToggleButton } from "../../toggle-button";
-import { ToggleButtonGroup } from "../../toggle-button-group";
+import { ToggleButton } from "../../../../core/src/toggle-button";
```
Comment (origami-z) [inline packages/lab/src/query-input/internal/QueryInputBody.tsx:13]: `@salt-ds/core`?

**Review by origami-z — COMMENTED** [review]
ToggleButtonGroup stories (not doc) remained in lab in storybook? https://2e6ea3b2.saltdesignsystem-storybook.pages.dev/?path=/story/lab-toggle-button-group--horizontal

File: .changeset/thin-candles-raise.md
Comment (alycrys) [inline .changeset/thin-candles-raise.md:1]: I think this belongs to another PR?

**Review by alycrys — APPROVED** [review]

**Review by Fercas123 — APPROVED** [review]

**Review by origami-z — APPROVED** [review]
